pkg/controllers/provisioningv2/machineconfigcleanup/controller.go

```diff
@@ -0,0 +1,237 @@
+package machineconfigcleanup
+
+import (
+	"context"
+	"fmt"
+
+	v3apis "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/rancher/pkg/controllers/capr/dynamicschema"
+	"github.com/rancher/rancher/pkg/fleet"
+	v3 "github.com/rancher/rancher/pkg/generated/controllers/management.cattle.io/v3"
+	image2 "github.com/rancher/rancher/pkg/image"
+	"github.com/rancher/rancher/pkg/settings"
+	"github.com/rancher/rancher/pkg/systemtemplate"
+	"github.com/rancher/rancher/pkg/wrangler"
+	"github.com/rancher/rancher/tests/v2/actions/kubeapi/rbac"
```

Comment:
Ah, we shouldn't be including this btw.

Reply:
oops. 
Doesn't look like there's a good place to read local cluster name from, used "local" directly as most pkgs have their own localClusterName defined and it was just used once here.